saleor/graphql/invoice/schema.py

```diff
@@ -0,0 +1,19 @@
+import graphene
+
+from .mutations import (
+    CreateInvoice,
+    DeleteInvoice,
+    RequestDeleteInvoice,
+    RequestInvoice,
+    SendInvoiceEmail,
+    UpdateInvoice,
+)
+
+
+class InvoiceMutations(graphene.ObjectType):
```

Comment:
Wouldn't be better to swap words places? 
`request_invoice` -> `invoice_request`
It would be easier to have all `invoice` mutation in one place in schema.graphql file. 
@maarcingebala  What do you think? Same flow is for `products`, `apps`, `tokens`, `variants` etc